feature-pack/server/src/main/resources/modules/org/infinispan/server/endpoint/slot/module.xml

```diff
@@ -40,6 +40,7 @@
       <module name="org.jboss.marshalling" services="import" slot="@infinispan.module.slot@"/>
       <module name="org.jboss.resteasy.resteasy-jaxrs" services="import" />
       <module name="org.jboss.security.negotiation" />
+      <module name="org.jboss.vfs"/>
```

Comment:
Is vfs still needed? I thought it was only used for examining the proto files inside the jar by the DUP, but that one no longer exists.

Reply:
left over...